ONSdigital/response-operations-ui
# Added additional changes based on Dinas feedback

# What and why?
Update to text based on Dinas input.

# How to test?
Activate new user
- Get to the verify account screen (after creating account and clicking link)
- put 1 letter in new password and 1 different letter in the confirm password
- Press 'activate account'

Forgot password
- Click forget password
- Enter email addess
- Click link forgotten password url link (in logs)
- put 1 letter in new password and 1 different letter in the confirm password
- Click 'Confirm'
- Enter same password as already used
- Click 'Confirm'

# Trello
https://trello.com/c/fsrNks6A/1861-bug-fix-error-text-on-verify-account-page-4-days

https://trello.com/c/TaT9H6oM/2309-bug-fix-error-text-on-reset-account-password-page
